idl/join_node.idl.hh

```diff
@@ -66,5 +66,6 @@ struct join_node_response_result {};
 verb [[ip]] join_node_query (raft::server_id dst_id, service::join_node_query_params) -> service::join_node_query_result;
 verb [[ip]] join_node_request (raft::server_id dst_id, service::join_node_request_params) -> service::join_node_request_result;
 verb join_node_response (raft::server_id dst_id, service::join_node_response_params) -> service::join_node_response_result;
+verb [[with_client_info, one_way, ip]] notify_banned (raft::server_id dst_id);
```

Comment:
This verb is unrelated to the join procedure. Isn't there a place better place for it?